src/main/java/graphql/execution/directives/DirectivesResolver.java

```diff
@@ -25,14 +26,14 @@ public class DirectivesResolver {
     public DirectivesResolver() {
     }
 
-    public Map<String, GraphQLDirective> resolveDirectives(List<Directive> directives, GraphQLSchema schema, Map<String, Object> variables, GraphQLContext graphQLContext, Locale locale) {
```

Comment:
I wonder if we should just return a `List<GraphQLDirecitve>`since we don't even use the map result. It's immediately discarded as we call `Map.values()`